app/javascript/packs/application.tsx

```diff
@@ -42,7 +42,9 @@ import * as Maintaining from '../components/maintaining'
 import * as Notifications from '../components/notifications'
 import * as Mentoring from '../components/mentoring'
 import * as Student from '../components/student'
+import { Submission } from '../components/student/Editor'
```

Comment:
@SleeplessByte I now have been importing a type from the component to the pack JS. What is a better way to do this?

Reply:
import type instead of import